pkg/orchestrator/etcd_worker.go

```diff
@@ -283,14 +283,33 @@ func (worker *EtcdWorker) cloneRawState() map[util.EtcdKey][]byte {
 	return ret
 }
 
+var maxBatchPatchSize = 96
+
 func (worker *EtcdWorker) applyPatchGroups(ctx context.Context, patchGroups [][]DataPatch) ([][]DataPatch, error) {
+	getBatchPatches := func(patchGroups [][]DataPatch) ([]DataPatch, int) {
+		batchPatches := patchGroups[0]
```

Comment:
check len before get patchGroups[0] ?

Reply:
In any condiction we should make sure a `patches` from `patchGroups` does not be splited.

Reply:
the caller will do this check.